apps/web/app/(app)/environments/[environmentId]/settings/lookandfeel/EditPlacement.tsx

```diff
@@ -17,35 +13,25 @@ const placements = [
   { name: "Centered Modal", value: "center", disabled: false },
 ];
 
-interface EditPlacementProps {
```

Comment:
These changes seem to have killed the product settings and leads to build errors. Please take a look and fix :-)